app/models/user.rb

```diff
@@ -40,6 +42,8 @@ class User < ApplicationRecord
   scope :search_user, -> role, data {where("role = ? AND (phone_number = ? OR
     name LIKE ?)", role, data, "%#{data}%")}
   scope :is_online, -> {where online: true}
+  scope :shipper, -> {where role: "Shipper"}
+  scope :shop, -> {where role: "Shop"}
 
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.Style/AlignParameters
**Severity**: INFO
**File**: app/models/user.rb L47

Align the parameters of a method call if they span more than one line.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*